app/styles/fonts.scss

```diff
@@ -0,0 +1,72 @@
+/* vietnamese */
```

Comment:
there is no need to add this file as long as you have the url of the font added to the html page